### DISCUSSION TITLE
obsidian 支持最新的 mac m1 芯片吗？

### DISCUSSION THREAD

**Discussion by noel-blazblue** [obsidianzh/forum#72] [General综合]
据说还得转义

**Comment by BlackFlame33** [comment]
![image](https://user-images.githubusercontent.com/38367403/112722342-e3102900-8f43-11eb-8d43-820bfd93883b.png)

架构应该是Apple的了，用着还是挺好，没有闪退过

**Comment by Quorafind** [comment]
有原生支持 M1 的版本，在官网上可以直接下载
